tests/test_server_basic.py

```diff
@@ -1810,7 +1810,9 @@ def test_posting_renku(dispatcher_live_fixture_with_renku_options, dispatcher_te
 
     extracted_api_code = DispatcherJobState.extract_api_code(session_id, job_id)
     token_pattern = r"(\'|\")token(\'|\"):.\s?(\'|\").*?(\'|\")"
```

Comment:
would this work? it's simpler

```suggestion
    token_pattern = r"[\'\"]token[\'\"]:\s*?[\'\"].*?[\'\"]"
```

Reply:
yes thanks!